app/src/test/java/com/example/android/architecture/blueprints/todoapp/data/DefaultTaskRepositoryTest.kt

```diff
@@ -103,16 +107,12 @@ class DefaultTaskRepositoryTest {
 
     @Test
     fun saveTask_savesToLocalAndRemote() = runTest {
-        // Make sure newTask is not in the remote or local datasources
-        assertThat(networkDataSource.tasks).doesNotContain(newTask.toNetwork())
-        assertThat(localDataSource.tasks).doesNotContain(newTask.toLocal())
-
         // When a task is saved to the tasks repository
-        val newTask = tasksRepository.createTask(newTask.title, newTask.description)
+        val newTaskId = tasksRepository.createTask(newTask.title, newTask.description)
 
-        // Then the remote and local sources are called
-        assertThat(networkDataSource.tasks).contains(newTask.toNetwork())
-        assertThat(localDataSource.tasks?.contains(newTask.toLocal()))
+        // Then the remote and local sources contain the new task
+        assertEquals(true, networkDataSource.tasks?.map { it.id }?.contains(newTaskId))
+        assertEquals(true, localDataSource.tasks?.map { it.id }?.contains(newTaskId))
```

Comment:
Some tests use `assertEquals(true, ...)` and some `assertThat(...).isTrue()`

Reply:
Good point. I will keep this as `assertThat`. 

Here's an issue to fix the consistency: https://github.com/android/architecture-samples/issues/932